Tokenomics and builders fixes (#82)
sora-xor/sora-docs: src/tbcd.md

# Token Bonding Curve Dollar (TBCD)

> - Token Bonding Curve Dollars (TBCD) is a new proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve
> - TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low
> - TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via on-chain referendum decided by all XOR token holders

Reviewer: ```suggestion
- Token Bonding Curve Dollars (TBCD) is a new proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve.
- TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low.
- TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via on-chain referendum decided by all XOR token holders.
```

## Why Implement Token Bonding Curve Dollars (TBCD)

> The SORA ecosystem has grown over the years and now there are many tokens and [eclectic](https://cerestoken.io) [builders](https://adar.com). With growth comes maturity, and SORA has recently been moving towards spinning out XSTUSD, a synthetic stablecoin, into the comprehensive SORA synthetics (XST) platform.

Reviewer: probably makes sense to link to the xst page

However, many of the partners we engage with for institutional projects are not ready to price everything in XOR, the native token of the SORA network, yet. Instead, they need a token that is readily convertible to XOR on-demand, yet pegged to the USD.

TBCD (**T**oken **B**onding **C**urve **D**ollars) aims to fill this void by being a token that is convertible to XOR, as a reserve asset of the SORA token bonding curve, while also being worth $1, because the token bonding curve always treats TBCD as having the value of $1 USD.

As you may recall, the [SORA token bonding

> curve](https://wiki.sora.org/token-bonding-curve) is a smart contract,

Reviewer: this a link to the old wiki, should be a relative link within this doc

built right into Polkaswap, which mints XOR in exchange for reserve
assets, and burns XOR to return the reserve assets. There is a margin
of 20% between buy and sell, which is used for buying back and burning
VAL, giving funds for future VAL DAOs, as well as to put into accounts
that in the future can be used for project funding, and giving
stipends to SORA citizens.

> ![The SORA Token Bonding Curve and uses of margin between buy and sell

Reviewer: re ` and uses of margin`: maybe `the uses`? or `and the use of`?

Author: uses of the margin is the most apropos

functions](/.gitbook/assets/flow-of-funds-new.png)
The SORA Token Bonding Curve and uses of margin between buy and sell

> functions

Reviewer: ```suggestion
```

if it's meant to be the description for the image, put it above the image

TBCD can only be created and allocated by on-chain governance, which
means that XOR token holders decide the supply. In this way, TBCD can
contribute to the productive economy of SORA because builders can be
funded in TBCD, which can lead to more uses for SORA ecosystem
tokens.
XOR is not diluted in purchasing power, because any minting of XOR is done only by giving TBCD to the token bonding curve, so TBCD becomes an asset that balances out any XOR put into circulation; in fact, the token bonding curve over-collateralizes, since there is a 20% margin between buy and sell, and part of that margin is XOR that goes into funds that are not currently distributed and put into circulation.

Because the SORA token is going to target a price point of around
$600, this means that TBCD will have little utility if XOR costs $600,
while the secondary market price is so much lower (currently it is

> $3).

Reviewer: it's a good idea to specify when is "currently": e.g. as of June 2023 or something
also: is it really $3?

Author: removed altogether

To make TBCD useful, instead of using the ~$600 price point, it will

> be able to buy XOR from the TBC at the current market price + $1. This

Reviewer: ```suggestion
be possible to buy XOR from the TBC at the current market price plus $1. This
```

is still higher than the market price, which will incentivize people
to avoid dumping TBCD for XOR when the XOR price is low (because they

> won’t get a full $1 of value). For example, if XOR is $3 then TBCD

Reviewer: ```suggestion
won’t get a full $1 of value). For example, if XOR is $3, then TBCD
```

which is not very appealing, but if XOR is at $100, then TBCD holders
can buy XOR for just $101, which is a negligible premium.
Eventually the premium can go away entirely, when XOR is at the nominal token bonding curve price.

As with other token bonding curve reserve assets, the sell price of
XOR for TBCD from the SORA token bonding curve will be 20% less than
the price for buying.

## Algorithmic Central Banking

TBCD is a great example showcasing the possibilities of the SORA token bonding curve to create monetary innovation for the SORA economy. TBCD solves both the question of how to fund the SORA economy’s productive expansion (the creation of new goods and services) and how to build up reserves in the token bonding curve to create a backing aimed at increasing the stability of XOR. By minting TBCD via on-chain referendum, builders can get the resources they need to buidl up the SORA economy and platform, while any TBCD they sell to the SORA token bonding curve will be owned by the SORA ecosystem as protocol-owned liquidity.

While not operating at the official token bonding curve price, TBCD
still acts as a form of “training wheels” for the SORA economy, where

> the token bonding curve can be activated and build up some assets,

Reviewer: ```suggestion
the token bonding curve can be activated and used to build up some assets,
```

> selling XOR via a primary market at a premium to the free-floating

Reviewer: ```suggestion
while selling XOR via a primary market at a premium to the free-floating
```
